core/kazoo_attachments/src/kz_att_http.erl

```diff
@@ -139,11 +139,15 @@ format_url(Fields, JObj, Args, Separator) ->
 format_url_field(JObj, Args, Fields, Acc)
   when is_list(Fields) ->
     [format_url(Fields, JObj, Args, <<>>) | Acc];
+format_url_field(JObj, Args, #{<<"arg">> := Arg}, Fields) ->
+    format_url_field(JObj, Args, {arg, Arg}, Fields);
```

Comment:
https://github.com/2600hz/kazoo/blob/master/core/kazoo_attachments/src/kz_att_http.erl#L29
```Erlang
Fields = maps:get('field_list', Params, default_format()),
```
Params is map, so this JSON in data plan will be converted to map and `field_list` will be parsed in `format_url_field/4`
```JSON
"attachments": {
       "modb-att-http": {
           "handler": "http",
           "settings": {
               "url": "http://ip:80",
               "verb": "put",
               "field_list": [
                   {
                       "field": "pvt_account_id"
                   },
                   {
                       "field": "owner_id"
                   },
                   {
                       "arg": "id"
                   },
                   {
                       "arg": "attachment"
                   }
               ]
           }
       }
...
```